apps/win32/wdio.conf.js

```diff
@@ -6,7 +6,7 @@ const appArgs = 'basePath ' + path.resolve('dist') + ' plugin defaultplugin bund
 const appDir = path.dirname(require.resolve('@office-iss/rex-win32/rex-win32.js'));
 
 const defaultWaitForTimeout = 20000;
-const defaultConnectionRetryTimeout = 20000;
+const defaultConnectionRetryTimeout = 60000;
```

Comment:
Is there a reason for the time increase?

Reply:
I've noticed the first test had to re-run a couple times over the past week due to a failure, so I think increasing this timer will give WDIO more time to connect to the app